pkg/alerts/nrql_conditions.go

```diff
@@ -565,6 +603,16 @@ const (
 		}
 		type
 		violationTimeLimit
+		expiration {
+      closeViolationsOnExpiration
```

Comment:
nit: fix indentation?

Reply:
Not sure I see what you're seeing 🤔

Reply:
Ah, it's my chrome extension thats hiding it. Must be a copy and paste thing.